### PR TITLE
Prepare release

### LINKED ISSUE #61: New release?
Hey 👋,

I wanted to use `flamegraph` for some tests which seems to have been added in #27 however I think the most recent release was cut before this PR was merged.

Could you release a new verison please?

### LINKED ISSUE #65: Broken "increase your team's skills" link in README.md

### LINKED ISSUE #66: New release
Is there a set of things that need to be implemented before the next release? Just asking cause I'm installing via git so I can get features not yet published to cargo i.e. `--bench`

### PR DESCRIPTION
**Pull request by jonas-schievink**
Closes #66 
Closes #61 
Closes #65

Will `cargo publish` once merged.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "flamegraph"
-version = "0.1.13"
+version = "0.2.0"
 authors = ["Tyler Neely <t@jujit.su>"]
 description = "A simple cargo subcommand for generating flamegraphs, using inferno under the hood"
 license = "MIT/Apache-2.0"
```
Comment (matklad) [inline Cargo.toml]: A more standards-conforming syntax is MIT OR Apache-2.0

File: Cargo.lock [hunk 2]
```diff
@@ -466,146 +492,149 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
...
+version = "0.4.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "proc-macro2 0.4.27 (registry+https://github.com/rust-lang/crates.io-index)",
- "quote 0.6.11 (registry+https://github.com/rust-lang/crates.io-index)",
- "syn 0.15.29 (registry+https://github.com/rust-lang/crates.io-index)",
- "unicode-xid 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "proc-macro2 1.0.8 (registry+https://github.com/rust-lang/crates.io-index)",
+ "quote 1.0.2 (registry+https://github.com/rust-lang/crates.io-index)",
+ "syn 1.0.14 (registry+https://github.com/rust-lang/crates.io-index)",
```
Comment (matklad) [inline Cargo.lock]: I think this is an old cargo.lock format, which is prone to conflicts. Might be a good idea to jus pt regenerate from scratch to get the new format.

**Comment by jonas-schievink** [comment]
Got CI working again. Bors should also be up.

bors ping

**Comment by jonas-schievink** [comment]
bors r=matklad

**Pull request merged**